pkg/manifests/tls.go

```diff
@@ -110,7 +112,9 @@ func RotateGRPCSecret(s *v1.Secret) error {
 		}
 	} else {
 		template := curCA.Config.Certs[0]
-		template.NotAfter = time.Now().Add(time.Duration(crypto.DefaultCertificateLifetimeInDays) * 24 * time.Hour)
+		now := time.Now()
+		template.NotBefore = now.Add(-1 * time.Second)
```

Comment:
Why can't we just use `noq`, does one second make so much difference?

Reply:
I've copied what's done in `github.com/openshift/library-go/pkg/crypto`:
https://github.com/openshift/cluster-monitoring-operator/blob/25720e264449ad870c1570b0a6674c545d7eac73/vendor/github.com/openshift/library-go/pkg/crypto/crypto.go#L911-L912

AFAICT it's very unlikely that we ever have a race by using `now`.

Reply:
Thanks! I am fine with either, was just curious about the reasoning.